domain/infosync/info.go

```diff
@@ -271,6 +279,44 @@ func (is *InfoSyncer) RemoveServerInfo() {
 	}
 }
 
+type topologyInfo struct {
+	ServerVersionInfo
+	StatusPort uint
```

Comment:
Why there is no json tags? Will it result in inconsistent field naming convension?

Reply:
No, I just fixed it.